.php_cs

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
I personally absolutely dislike the rule of using 4 spaces instead of tabs … but I understand the point of sticking to the rules … only I don’t see the point in in this rule: 4 spaces are harder to edit than tabs and I can’t configure the indent size as I like it …. but then that’s a pointless political discussion ;)

However, does it really need that extra file just for that one-time change?